code/modules/hydroponics/plant_genes.dm

```diff
@@ -109,10 +109,10 @@
 
 /datum/plant_gene/reagent/get_name()
 	var/formatted_name
-	if(mutability == PLANT_GENE_IMMUTABLE)
-		formatted_name += "Immutable "
-	else if(mutability == PLANT_GENE_REMOVABLE)
-		formatted_name += "Excisable "
+//	if(mutability_flags & PLANT_GENE_IMMUTABLE)
+//		formatted_name += "Immutable "
+//	else if(mutability_flags & PLANT_GENE_REMOVABLE)
+//		formatted_name += "Excisable "
```

Comment:
commented code

Reply:
I think I fixed that 4 days ago, or am I going insane?